packages/nextjs/app/builders/0x62CeF3Ca8b52a9C69a17236CA2c56Cdb7a383E8e/page.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+import { Github, Linkedin, Twitter } from "lucide-react";
```

Comment:
I'm receiving a deprecation warning for lucide-react. You may want to consider exploring an alternative library.